packages/atomic/src/components/common/atomic-focus-trap.tsx

```diff
@@ -0,0 +1,112 @@
+import {Component, Element, Listen, Prop, Watch} from '@stencil/core';
```

Comment:
I wasn't sure in which folder to put this component, since the other components in `common` are functional components. Do you think that's the correct place?

Reply:
I mean... I don't remember having ever worked on a software project that did not have that kind of "put your code here if you are not sure" type of folder 😂.

So 🤷‍♂️, yeah it's fine IMO.